tools/github-actions/new-version/readme.md

```diff
@@ -0,0 +1,42 @@
+# New version action
+
+## Overview
+
+This task computes new version candidates based on the repository tags as well as the branch on which it is executed.
+It supports in terms of branching model:
+* Release branches named ``release/Major.Minor[.0-alpha|beta|next|rc]``
```

Comment:
This is actually based on the `releaseBranchRegExp` input and support more than this pattern (but this is the default pattern)